.github/workflows/pr_testing.yaml

```diff
@@ -42,6 +42,21 @@ jobs:
       - name: Check whether REFERENCE.md has uncommitted changes
         run: git diff --exit-code REFERENCE.md
 
+  specs:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: '3.3'
+          bundler-cache: true
+      - name: Ensure we have both wget and curl for testing
+        run: |-
+          sudo apt update
+          sudo apt install -y wget curl
```

Comment:
I would have assumed that GitHub installs that by default

Reply:
Probably, this just doesn't leave it to chance.

Reply:
Although looking at what ci.yaml does, running specs here again is redundant, so I'll remove this section from pr_testing.yaml

Reply:
Ok, that section is gone.